be/src/storage/async_delta_writer.cpp

```diff
@@ -1,9 +1,8 @@
 // This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
 
-#include "storage/vectorized/async_delta_writer.h"
-
-#include <fmt/format.h>
+#include "async_delta_writer.h"
```

Comment:
we make the `be/src` as one search path, better to use `#include "storage/async_delta_writer.h"`